packages/cysync-core/src/dialogs/Send/Dialogs/Components/FeeSection/index.tsx

```diff
@@ -78,12 +93,27 @@ export const FeeSection: React.FC = () => {
     solana: () => ({}),
   };
 
-  const getFeeInputComponent = (coinFamily: CoinFamily) => {
+  const getFeeInputComponent = () => {
+    if (!selectedAccount) return null;
+    const coinFamily = selectedAccount.familyId as CoinFamily;
     const Component = feeInputMap[coinFamily];
     const props = feeInputPropsMap[coinFamily]();
     return <Component {...props} />;
   };
 
+  const getFeeHeaderComponent = () => {
+    if (!selectedAccount) return null;
+    const coinFamily = selectedAccount.familyId as CoinFamily;
+    const Component = feeHeaderMap[coinFamily](selectedAccount.assetId);
```

Comment:
Use parent asset ID, otherwise it won't work for tokens
```suggestion
    const Component = feeHeaderMap[coinFamily](selectedAccount.parentAssetId);
```